README.md

```diff
@@ -24,15 +24,15 @@ it is not mandatory. Once setup, run
 
 ```bash
 pip install -r requirements.txt
-pip install -r requirements_additional.txt
 ```
 
 You can then clone or download the scoring system. Once cloned or
 downloaded, go inside the ```ismrm_2015_tractography_challenge_scoring```
 directory, and run
 
 ```bash
-python setup.py build_all
+python setup.py build_ext --inplace
+python setup.py install
```

Comment:
Putting it here, but you don't need the `export PYTHONPATH` code now with the install in a venv.